app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> int:
```

Comment:
Provide more detailed annotations for the team parameter, specifying it as a list of Player objects.

Do the same for `elves_concert `and `feast_of_the_dwarves` functions